### DISCUSSION TITLE
Script to AIM AUTOMATICALLY

### DISCUSSION THREAD

**Discussion by guguxh** [wxyz-abcd/node-haxball#54] [General]
This is possible? How I calculate the times the character need to press keys to aim in X place?

**Comment by wxyz-abcd** [comment]
Predicting the future is hard if not impossible, because we cannot know what the player inputs will be. However, it's possible to show some indicator(a direct line or arrow) to show the direction the ball will go when you hit the ball at each render cycle. This does not sound hard. I might work on it some time. Thanks for suggestion.
